newrelic/exporter.go

```diff
@@ -53,34 +63,109 @@ func NewExporter(serviceName, apiKey string, options ...func(*telemetry.Config))
 	}
 	return &Exporter{
 		harvester:   h,
-		serviceName: serviceName,
+		serviceName: service,
 	}, nil
 }
 
+// NewExportPipeline creates a new OpenTelemetry telemetry pipeline using a
+// New Relic Exporter configured with default setting. It is the callers
+// responsibility to stop the returned push Controller. This function uses the
+// following environment variables to configure the exporter installed in the
+// pipeline:
+//
+//    * `NEW_RELIC_API_KEY`: New Relic API key.
```

Comment:
In my experience there can be confusion around API key vs. license key - it might be worth being even more explicit about that in this documentation.

Reply:
Added more docs. Please let me know your suggestion if you think more is needed.